fase_di_sviluppo/src/modules/duration.js

```diff
@@ -5,7 +5,7 @@
  * extension of the class Date.
  * 
  * @file This file defines the Duration class.
- * @author Andrea Lanzani
+ * @author Andrea Lanzani, Fabio Casetta
```

Comment:
Gli autori dovrebbero essere su linee separate
```suggestion
 * @author Andrea Lanzani
 * @author Fabio Casetta
```